scanpointgenerator/generators/lissajousgenerator.py

```diff
@@ -9,27 +9,27 @@
 class LissajousGenerator(Generator):
     """Generate the points of a Lissajous curve"""
 
-    def __init__(self, names, units, box, num_lobes,
-            num_points=None, alternate_direction=False):
+    def __init__(self, axes, units, box, num_lobes,
+            size=None, alternate=False):
         """
         Args:
-            names (list(str)): The scannable names e.g. ["x", "y"]
-            units (str): The scannable units e.g. "mm"
+            axes (list(str)): The scannable axes e.g. ["x", "y"]
+            units (list(str)): The scannable units e.g. ["mm", "mm"]
             box(dict): Dictionary of centre, width and height representing
```

Comment:
box dict now looks inconsistent with units, axes, etc. Can we have centre and span 2 element lists instead please?